python/tvm/meta_schedule/testing/torchbench/utils.py

```diff
@@ -0,0 +1,72 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+import os
+import sys
+from dataclasses import dataclass
+
+import torch
+
+
+def find_torchdynamo():
+    candidates = [
+        "torchdynamo",
+        "../torchdynamo",
+        "../../torchdynamo",
+    ]
+    for library_dir in candidates:
+        if os.path.exists(f"{library_dir}/benchmarks"):
+            return library_dir
+    else:
+        raise RuntimeError(
+            f"Cannot find directory for torchdynamo. You need to clone https://github.com/pytorch/torchdynamo to the parent directory of cwd."
+        )
+
+
+DYNAMO_DIR = find_torchdynamo()
+sys.path.append(DYNAMO_DIR)
+sys.path.append(f"{DYNAMO_DIR}/benchmarks")
+
+from benchmarks.common import same, timed
+from torchbench import TorchBenchmarkRunner
+
+
+def load_torchdynamo_benchmark_runner(is_cuda: bool, cosine_similarity: bool = False):
+    @dataclass
+    class RunnerArgs:
+        ci: bool = False
```

Comment:
A brief comment explaining these args are for torch dynamo runner configuration would be apprecidated.